### ISSUE TITLE
adaptor fasta

### ISSUE THREAD

**Issue by Yuling-Zhao** [fl-yu/CUT-RUNTools-2.0#46]
Dear all,

I couldn't find the adaptors of my samples in the adaptor directory. I used NEBNext® Multiplex Oligos for Illumina. Does it require a new fasta?

Thanks for your help in advance!

Best,
Yuling

**Comment by fl-yu** [comment]
Hi Yuling, we currently only support Truseq or Nextera adapters. For more details please refer to https://github.com/fl-yu/CUT-RUNTools-2.0/issues/24.

I am not familiar with that Oligos, but  I guess you will probably need to contact those people to get such adaptor sequences.

**Comment by Yuling-Zhao** [comment]
Thanks for the reference. May I ask how to generate the fasta file for adaptor sequence? I have the document for the sequence.

Best,
Yuling

**Comment by Yuling-Zhao** [comment]
https://international.neb.com/-/media/nebus/files/manuals/manuale6440.pdf?rev=903fc5e904cc4b2b9843b3b1f7d27f02&hash=7E9F7AC01683A3D41DFF5CEC33B6C156

This page contains the adaptor sequences for this kit, if you have interest.

Thanks!

**Comment by fl-yu** [comment]
Good to know you got the adaptor sequences working. I will close this issue and please feel free to reopen it once have further issues with the adaptor trimming.

**Comment by ChristianRohde** [comment]
Hi,

I also end in this issue: we also use NEBNext Oligos for library prep and sure we know the sequence and can provide a fasta files withe these based on the information of the pdf. Unfortunately I do not understand how to add this fasta to your pipeline in a way that the correct oligos for PE trimming will be selected. Can you please explain how this can be done? 

Thank you,
Christian

**Comment by ChristianRohde** [comment]
Hi, I apologize that I was asking for support too early: I learned from the NEB website that basically 'The NEBNext libraries for Illumina resemble TruSeq libraries and can be trimmed like TruSeq:'

https://www.neb.com/en/faqs/2021/01/15/what-sequences-need-to-be-trimmed-for-nebnext-libraries-that-are-sequenced-on-an-illumina-instrument 

I found the sequence of the oligos listed there in your and trimmomatic adapter sequences https://github.com/usadellab/Trimmomatic/blob/main/adapters/TruSeq3-PE-2.fa listed as:

>PE1_rc
AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA
>PE2_rc
AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC

The only difference is that the oligos listed at NEB are one nucleotide shorter. While fastqc indeed showed little amount of Illumina Universal Adapter in the original fastqc, this was removed after trimming with your pipeline. I hope this comment helps.

Best,
Christian
